inst/examples/Button3.R

```diff
@@ -0,0 +1,47 @@
+
+# Example 3
+library(shiny)
+library(shiny.fluent)
+library(shinyjs)
+
+# This example app shows how to use a Fluent UI Button to trigger a file upload.
+# File upload is not natively supported by shiny.fluent, so shinyjs is used
+# to trigger the file upload input.
+ui <- fluentPage(
+  useShinyjs(),
+  Stack(
+    tokens = list(
+      childrenGap = 10L
+    ),
+    horizontal = TRUE,
+    DefaultButton.shinyInput(
+      inputId = "uploadFileButton",
+      text = "Upload File",
+      iconProps = list(iconName = "Upload")
+    ),
+    div(
+      style = "
+        visibility: hidden;
+        height: 0;
+        width: 0;
+      ",
+      fileInput(
+        inputId = "uploadFile",
+        label = NULL
+      )
+    )
+  ),
+  textOutput("file_path")
+)
+
+server <- function(input, output) {
+  observeEvent(input$uploadFileButton, {
+    click("uploadFile")
+  })
+
+  output$file_path <- renderPrint({
+    input$uploadFile
+  })
```

Comment:
Seems to render more than just the file path:
![Screenshot from 2023-06-15 14-21-50](https://github.com/Appsilon/shiny.fluent/assets/10000651/ab4c32e1-37bf-4374-a3fa-3bf3e2931a23)